command/git_push.go

```diff
@@ -24,6 +25,7 @@ type gitPush struct {
 	DryRun         bool   `yaml:"dry_run" mapstructure:"dry_run"`
 	CommitterName  string `yaml:"committer_name" mapstructure:"committer_name"`
 	CommitterEmail string `yaml:"committer_email" mapstructure:"committer_email"`
+	Token          string `plugin:"expand" mapstructure:"token"`
```

Comment:
Explicitly set a yaml tag for this

Reply:
done.
Why is this important? Would someone put their token as a parameter in their config file?

Reply:
Isn't that why you added this parameter? To allow users to override the global token if needed?

Reply:
my thought was if a token were a project variable it would get expanded and override the global token.

Reply:
That was my thought too. How would it be passed in if not in the config file?

Reply:
I get it now (@Kimchelly explained it to me). The token is passed as a parameter in the config file, but could just be an expansion of a project level variable. 
I had previously been envisioning a project level variable called "token" that ends up in the task's expansions.